tests/test_hosting_commands.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from syncano_cli.hosting.utils import HostingCommands
+from syncano_cli.main import cli
+from tests.base import BaseCLITest
+
+
+class HostingCommandsTest(BaseCLITest):
```

Comment:
I guess we could remove the publish_files test as test_file_list is kind of depending on that functionality anyway